main.tf

```diff
@@ -169,8 +171,8 @@ resource "aws_cognito_identity_provider" "apple" {
   provider_details = {
     client_id        = var.apple_service_id
     team_id          = var.apple_team_id
-    key_id           = var.apple_key_id
-    private_key      = var.apple_private_key_path
+    # key_id           = var.apple_key_id       # This value is mandatory when enabling SignInWithApple for the Cognito user pool.
+    # private_key      = var.apple_private_key  # This value is mandatory and specifies the path to the Apple private key for SignInWithApple.
```

Comment:
Apple login shpold be optional, no deleted because the compatibility with old versions will be broken